test-app/tsconfig.json

```diff
@@ -1,58 +1,16 @@
 {
+  "extends": "@tsconfig/ember/tsconfig.json",
   "compilerOptions": {
-    "target": "es2019",
-    "allowJs": true,
-    "moduleResolution": "node",
-
-    // Don't be viral!
-    "allowSyntheticDefaultImports": false,
-    "esModuleInterop": false,
-
-    // strictness
-    "strict": true,
-    "noUncheckedIndexedAccess": true,
-    "exactOptionalPropertyTypes": true,
-
-    // lint-style rules
-    "noFallthroughCasesInSwitch": true,
-    "noUnusedLocals": true,
-    "noUnusedParameters": true,
-    "noImplicitReturns": true,
-
-    "noEmitOnError": false,
-    "noEmit": true,
-    "inlineSourceMap": true,
-    "inlineSources": true,
     "baseUrl": ".",
-    "module": "es6",
-    "experimentalDecorators": true,
     "paths": {
-      "test-app/tests/*": [
-        "tests/*"
-      ],
-      "test-app/*": [
-        "app/*"
-      ],
-      "tracked-built-ins": [
-        "../addon/addon"
-      ],
-      "tracked-built-ins/*": [
-        "../addon/addon/*"
-      ],
-      "tracked-built-ins/test-support": [
-        "../addon/addon-test-support"
-      ],
-      "tracked-built-ins/test-support/*": [
-        "../addon/addon-test-support/*"
-      ],
-      "*": [
-        "types/*"
-      ]
+      "test-app/tests/*": ["tests/*"],
+      "test-app/*": ["app/*"],
+      "tracked-built-ins": ["../addon/addon"],
```

Comment:
side-note: I need to check if these mappings for `tracked-built-ins` is actually needed. TS should just be happy in test-app as `tracked-built-ins` is now injected like regular npm package and has all the types in it.

Reply:
Ah, yep, we should be able to remove this.